### ISSUE TITLE
Allow to choose colour for prompt items

### ISSUE THREAD

**Issue by karthikk9828** [k-yokoishi/zsh-prompt-generator#26]
It would be great if we can get the script and the GUI for changing colours of each part of the prompt text.

**Comment by k-yokoishi** [comment]
Thank you for your comment!:smile:
Is this suggestion same as #21?

**Comment by karthikk9828** [comment]
Yes, but I would request to be able to select colour for each component.
